### ISSUE TITLE
Review new federal IT Accessibility police for relevance and share with ONRR

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#2701]
https://www.section508.gov/manage/policy-framework/introduction/

**Comment by lpgoldstein** [comment]
We are reaching out to the community to introduce a new best practice resource for agencies. The Government-wide IT Accessibility program has created a new IT Accessibility Policy Framework for assessing agency policies for accessibility information.

Currently, agencies lack a standard framework to comprehensively identify which policies should have accessibility information, if they actually have that information and if not, a means to update their policies accordingly. Without such a holistic system, we are likely to have a patchwork of policies with wide variation in how and to what extent they address information and communication technology (ICT) accessibility. 

Having the right amount of accessibility information in relevant policies is important so that employees have a reference as to how they should include accessibility in their tasks. If it is laid out in policy, employees:

Know that digital accessibility is important to leadership and to the agency in general

Understand their role in making ICT accessible in whatever business function the policy covers

Have an agency approved document to refer to in order to ensure they are following the correct procedures with respect to digital accessibility

The policy framework allows users to assess policy for its importance to digital accessibility and the sufficiency of accessibility information it contains. Using these metrics, users can determine the need or priority for updating a document with the additional accessibility information.

The framework is intended to be a helpful tool to improve accessibility across an organization, use is not mandatory. Please see attached a two-page document that further introduces the policy framework. Please reach out to the Government-wide IT Accessibility team at section.508@gsa.gov if you have any questions on how to use the framework.
